thewalrus/quantum/__init__.py

```diff
@@ -0,0 +1,202 @@
+    state_vector
+    density_matrix_element
+    density_matrix
+    fock_tensor
+    probabilities
+    update_probabilities_with_loss
+    update_probabilities_with_noise
+    normal_ordered_expectation
+    fidelity
+
+Details
+^^^^^^^
+
+.. autofunction::
+    pure_state_amplitude
+
+.. autofunction::
+    state_vector
+
+.. autofunction::
+    density_matrix_element
+
+.. autofunction::
+    density_matrix
+
+.. autofunction::
+    fock_tensor
+
+.. autofunction::
+    probabilities
+
+.. autofunction::
+    update_probabilities_with_loss
+
+.. autofunction::
+    update_probabilities_with_noise
+
+.. autofunction::
+    normal_ordered_expectation
+
+.. autofunction::
+    fidelity
+
+
+Utility functions
+-----------------
+
+.. autosummary::
+
+    reduced_gaussian
+    Xmat
+    Qmat
+    Covmat
+    Amat
+    complex_to_real_displacements
+    real_to_complex_displacements
+    find_scaling_adjacency_matrix
+    mean_number_of_clicks
+    mean_number_of_clicks_graph
+    find_scaling_adjacency_matrix_torontonian
+    gen_Qmat_from_graph
+    photon_number_mean
+    photon_number_mean_vector
+    photon_number_covar
+    photon_number_covmat
+    is_valid_cov
+    is_pure_cov
+    is_classical_cov
+    find_classical_subsystem
+    pure_state_distribution
+
+
+Details
+^^^^^^^
+"""
+
+from .fock_tensors import (
+    pure_state_amplitude,
+    state_vector,
+    density_matrix_element,
+    density_matrix,
+    fock_tensor,
+    probabilities,
+    loss_mat,
+    update_probabilities_with_loss,
+    update_probabilities_with_noise,
+    find_classical_subsystem,
+    _prefactor,
+)
+
+from .adjacency_matrices import (
+    find_scaling_adjacency_matrix,
+    find_scaling_adjacency_matrix_torontonian,
+    gen_Qmat_from_graph,
+)
+
+from .gaussian_state_tests import (
+    is_valid_cov,
+    is_pure_cov,
+    is_classical_cov,
+    fidelity,
+)
+
+from .conversions import (
+    reduced_gaussian,
+
+    Xmat,
+    Qmat,
+    Covmat,
+    Amat,
+    complex_to_real_displacements,
+    real_to_complex_displacements,
+)
+
+from .means_and_variances import (
+    photon_number_mean,
+    photon_number_mean_vector,
+    photon_number_covar,
+    photon_number_covmat,
+    photon_number_expectation,
+    photon_number_squared_expectation,
+    normal_ordered_expectation,
+    mean_number_of_clicks,
+    variance_number_of_clicks,
+    mean_number_of_clicks_graph,
+)
+
+from .photon_number_distributions import (
+    pure_state_distribution,
+    _squeezed_state_distribution,
+    _convolve_squeezed_state_distribution,
+)
+
+__all__ = [
+    "pure_state_amplitude",
+    "state_vector",
+    "density_matrix_element",
+    "density_matrix",
+    "fock_tensor",
+    "probabilities",
+    "update_probabilities_with_loss",
+    "update_probabilities_with_noise",
+    "normal_ordered_expectation",
+    "fidelity",
+]
+
+
+# old names for functions; remove in due time
+Means = real_to_complex_displacements
+Beta = complex_to_real_displacements
```

Comment:
These new names are great, in that they describe exactly what the function does :+1:

Stylistically, I have to admit though that I'm always personally biased towards shorter/more concise names where possible. But I can't think of any good candidates 🤔 

Similar to how NumPy has `np.linalg`, `svd` `eigvals`, `inv`, etc. Short and concise, but also descriptive enough so that you know what the functions do if you have a good understanding of linear algebra

Reply:
Maybe @nquesada has some ideas? But this is a minor point.

Reply:
I have to say that this is the best that we could come up with. The only thing that I could see being dropped is the `displacements` part, but other than that not sure how to make short.